src/main/java/uk/gov/hmcts/reform/wataskmonitor/services/jobs/initiation/InitiationTaskAttributesMapper.java

```diff
@@ -53,24 +53,30 @@ public InitiationTaskAttributesMapper(ObjectMapper objectMapper) {
         String assignee = camundaTask.getAssignee();
         String description = camundaTask.getDescription();
         // Local Variables
-        String type = getVariableValue(variables.get(TASK_TYPE.value()), String.class);
+        String type = getVariableValue(variables.get(TASK_TYPE.value()), String.class, null);
         CFTTaskState taskState = extractTaskState(variables);
-        String executionTypeName = getVariableValue(variables.get(EXECUTION_TYPE.value()), String.class);
-        String securityClassification = getVariableValue(variables.get(SECURITY_CLASSIFICATION.value()), String.class);
-        String taskTitle = getVariableValue(variables.get(TITLE.value()), String.class);
-
-        boolean autoAssigned = getVariableValue(variables.get(AUTO_ASSIGNED.value()), Boolean.class);
-        String taskSystem = getVariableValue(variables.get(TASK_SYSTEM.value()), String.class);
-        String jurisdiction = getVariableValue(variables.get(JURISDICTION.value()), String.class);
-        String region = getVariableValue(variables.get(REGION.value()), String.class);
-        String location = getVariableValue(variables.get(LOCATION.value()), String.class);
-        String locationName = getVariableValue(variables.get(LOCATION_NAME.value()), String.class);
-        String caseTypeId = getVariableValue(variables.get(CASE_TYPE_ID.value()), String.class);
-        String caseId = getVariableValue(variables.get(CASE_ID.value()), String.class);
-        String caseName = getVariableValue(variables.get(CASE_NAME.value()), String.class);
-        Boolean hasWarnings = getVariableValue(variables.get(HAS_WARNINGS.value()), Boolean.class);
-        String warningList = getVariableValue(variables.get(WARNING_LIST.value()), String.class);
-        String caseManagementCategory = getVariableValue(variables.get(CASE_MANAGEMENT_CATEGORY.value()), String.class);
+        String executionTypeName = getVariableValue(variables.get(EXECUTION_TYPE.value()),
+            String.class,
+            null);
+        String securityClassification = getVariableValue(variables.get(SECURITY_CLASSIFICATION.value()),
+            String.class,
+            null);
+        String taskTitle = getVariableValue(variables.get(TITLE.value()), String.class,null);
+
+        boolean autoAssigned = getVariableValue(variables.get(AUTO_ASSIGNED.value()), Boolean.class, false);
+        String taskSystem = getVariableValue(variables.get(TASK_SYSTEM.value()), String.class, null);
```

Comment:
It wasn't your code and probably better to address in another PR.  I will just leave a comment for reference. Do we need to return `null` here?  Better to avoid `null` everywhere and certainly we should try and not create new ones ourselves.